### PR TITLE
added files for /contact-us

### PR DESCRIPTION
**Pull request by AWBeckstrom**
Contact us page created with form inset. One additonal change will need to be made to this file, yet it is good to have the progress achieved committed. 
Attached picture reveals achieved results follow FIGMA example closely.
Additionally, a specialized font was inserted into the index.css file.
pr snapbody

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,6 +1,17 @@
-<!doctype html>
+<!DOCTYPE html>
```
Comment (jmedina1329) [inline index.html:1]: Why are you making changes to this file? What do these changes do?

Comment (AWBeckstrom) [reply index.html:1]: The link below gave me instructions of how to embed the font into the app.
https://fonts.google.com/selection/embed
That is why I made any changes in the index.html or index.css files

**Review by jmedina1329 — APPROVED** [review]

File: src/components/contact/ContactForm.jsx [hunk 2]
```diff
@@ -1,29 +1,58 @@
...
 function ContactForm() {
   return (
     <Fragment>
-      <Card style={{ width: "18rem" }}>
-        <Card.Body>
-          <Form>
-            <Form.Group className="mb-3" controlId="contactForm.NameInput">
-              <Form.Label>Name</Form.Label>
-              <Form.Control type="name" placeholder="your full name here" />
+      <Card style={{ width: "55rem" }} className={styles.card}>
```
Comment (jjcanell77) [inline src/components/contact/ContactForm.jsx]: Maybe add the style to the styles.card?

File: src/index.css [hunk 3]
```diff
@@ -28,27 +29,42 @@ body {
   place-items: center;
   min-width: 320px;
   min-height: 100vh;
+  font-family:"Merriweather", serif;
+  font-style: normal;
+  font-weight: 400;
 }
 
 h1 {
```
Comment (jjcanell77) [inline src/index.css:40]: This will affect all h1 classes. Maybe this should be added to your module.css instead

**Review by jjcanell77 — CHANGES_REQUESTED** [review]
Looks good, I'm just worried about the changes made to index.css as it will affect all other components

**Review by kristin-serafini — APPROVED** [review]

File: src/index.css [hunk 4]
```diff
@@ -65,10 +69,56 @@ button:focus-visible {
...
+  font-family: "Merriweather", serif;
+  font-weight: 900;
+  font-style: italic;
 }
 
 /* mapcontainer */
 .leaflet-container {
   height: 600px;
-  width: 100%;
-}
```
Comment (jjcanell77) [inline src/index.css]: Why did you delete this? " }"

File: src/components/contact/Contact.module.css [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    font-family: "Merriweather", serif;
+    width: 50%;
+}
+
+.contactFormTitle {
+    padding-bottom: 100px;  
+    padding-top: 100px;  
+    column-width: 45%;
+    text-align: left;
+    font-family: "Merriweather", serif;
```
Comment (jjcanell77) [inline src/components/contact/Contact.module.css]: You added this to all "h1"s and this class is on an "h1"  So you do not need this font family twice.

File: src/components/contact/Contact.module.css [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+ }
+
+.formHeader {
+    text-align: center;
+    font-family: "Merriweather", serif;
+    font-weight: 500;
+    color: rgb(3, 40, 3);
+}
```
Comment (jjcanell77) [inline src/components/contact/Contact.module.css]: Same here you use this on an h1 that you added to index.css but you have font family again.` If you were changing everything you added to h1 why not use a div instead ?

File: src/components/contact/Contact.module.css [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+
+.emailHQAddress {
+    padding-top: 2%;
+    font-family: "Merriweather", serif;
+}
+
+.phone {
+    padding-top: 5%;
+    font-family: "Merriweather", serif;
```
Comment (jjcanell77) [inline src/components/contact/Contact.module.css]: this is also in an h1 so you don't need the font family

File: src/components/contact/Contact.module.css [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+.phoneHQNumber {
+    padding-top: 2%;
+    font-family: "Merriweather", serif;
+    font-size: 1.5rem;
+}
+
+.follow {
+    padding-top: 5%;
+    font-family: "Merriweather", serif;
```
Comment (jjcanell77) [inline src/components/contact/Contact.module.css]: this is also in an h1 so you don't need the font family

**Review by jjcanell77 — CHANGES_REQUESTED** [review]

**Comment by jmedina1329** [comment]
Please update this branch with main

**Review by BWFischer — APPROVED** [review]

**Comment by BWFischer** [comment]
This branch has conflicts that must be resolved.  Please resolve these and resubmit this PR

**Review by KenStephens — APPROVED** [review]

**Review by jjcanell77 — APPROVED** [review]
